fileserver/crypto.py

```diff
@@ -1,6 +1,7 @@
 import nacl.public
 import os
-import pyonionreq.junk
+import pyonionreq
+from pyonionreq import junk
```

Comment:
We should get rid of this dependency here: `onionreq` was a half-baked helper lib that has been surpassed by functionality in `python3-session-util` (which are (some) python bindings for libsession-util).

The replacement is quite simple: this is the commit that updated SOGS: https://github.com/oxen-io/session-pysogs/pull/216/commits/7eecddd14611cfb82f03dcfc9e1628ce14d92727 (although the `import` there is missing a line: it should be `from session_util.onionreq import OnionReqParser`)

Reply:
I tried to swap this out (have all the changes locally) but it ran into the below issue - did I need to add `libsession-util` as a submodule or dependency as well somehow? (I couldn't see this being done in the PySOGS branch so wasn't sure how it should be done 😕)
```
The following packages have unmet dependencies:
  python3-session-util : Depends: libsession-util1.1.0 but it is not installable
```

Reply:
This has been updated now